jmh.gradle

```diff
@@ -17,7 +17,14 @@
  * under the License.
  */
 
-def jmhProjects = [ project("iceberg-spark2") ]
+def jmhProjects
```

Comment:
The formatting here is a bit odd but other than this, looks good to me

Reply:
Yeah. I could use `deef jmhProjects = []`, but IntelliJ complains about unused variable.

I then tried to be a bit more Scala like, and did `def jmhProjects = { ... return array_of_what_i_want }` but that generated a closure.

Basically, the best I think I can offer is `def jmhProjects = []` and then leave it like that.

Reply:
Discussed offline and changed it to be initialized with an array for readability.